src/20_Components/amp-live-list.html

```diff
@@ -0,0 +1,147 @@
+
+  This example shows an implementation with a server rendering a series of 10 elements, `amp-live-list` will poll the page every 15 seconds. Every time `amp-live-list` polls the page, the server adds a new item to the page. Refer to [server.go](https://github.com/ampproject/amp-by-example/blob/master/server.go) for implementation details.
+
+  When the user lands to this page the first time, the server returns one item and creates a cookie containing the number of times the user has accessed the page (it will be 1 on creation). For every other access, this number is incremented by one and a new item is added to the page server side. 
+  When the number is equal to 10, the server stops generating new content. If you want to see the demo again, delete the cookie or access the page using an Incognito tab.
+
+-->
+<!-- -->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <!-- #### Setup -->
+  <!--
+  Import the `amp-live-list` component in the header.
+
+  Use `amp-live-list-item` to style the children of the `amp-live-list`. Refer to the [documentation](https://github.com/ampproject/amphtml/blob/f2981e72843c8e71a9cd2535282e4c4947bb6882/extensions/amp-live-list/amp-live-list.md) for all the details about styling.
+  -->
+  <script async custom-element="amp-live-list" src="https://cdn.ampproject.org/v0/amp-live-list-0.1.js"></script>
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <style amp-custom>
+    amp-live-list > [items] {
+      display: flex;
+      flex-direction: column;
+    }
+
+    .amp-live-list-item {
+      background-color: #f8f8f8;
+      border-bottom: 1px solid rgba(0, 0, 0, .12);
+    }
+
+    .card-list {
+      display: flex;
+      margin: 15px;
+      padding: 10px;
+      align-items: center;
+      background-color: #fff;
+      box-shadow: rgba(0, 0, 0, .3) 0 0 5px;
+    }
+
+    .content {
+      font-size: 1.5rem;
+      margin-left: 15px;
+    }
+
+    .button {
```

Comment:
styling a `button` element with a class `.button` is a bit redundant. We should need to style buttons for each example but instead create a generic button style.